docs/content/operator/common/monitoring.mdx

```diff
@@ -1,6 +1,39 @@
 After a successful setup of a node, the next important step is to set up monitoring for the node.
 Monitoring is important to keep track of the node's health and performance.
 
+
+## Full Node Monitoring
+
+### Test JSON RPC Interface
+After the full node starts, you can test the [JSON-RPC interfaces](https://docs.iota.org/iota-api-ref).
+
+### View Activity on Your Local Full Node with IOTA Explorer
+
+The IOTA Explorer supports connecting to any network as long as it has `https` enabled. To view activity on your local full node, open the URL: [https://explorer.rebased.iota.org/](https://explorer.rebased.iota.org/), and select `Custom RPC URL` from the network dropdown in the top right.
+
+### View Resource Usage (CPU and Memory)
```

Comment:
I think we don't need this section, it's just basic commands from the system and docker. 👀 
Same goes to the `Viewing Logs` section.

Reply:
Kinda agree for the resource usage section (removed it) but can we keep the viewing logs section ? Just thinking now the setup is done by a script, users know a little bit less about what's been done 
So I wouldn't want users to have to guess the params even if the command seems basic to us